dist/index.html

```diff
@@ -12,7 +12,11 @@
 
         <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
-        <script src="js/vendor/modernizr-2.8.3.min.js"></script>
+        <script src="//cdnjs.cloudflare.com/ajax/libs/modernizr/2.8.3/modernizr.min.js"></script>
```

Comment:
We use a custom build of Modernizr, and want to enable people to do so. That's why we don't use a cdn for this. Also, you might want to read this blogpost: http://modernizr.com/news/modernizr-and-cdns/.